Application/Client Side/client.py

```diff
@@ -1,12 +1,12 @@
 """
-This is the client-side coding and handles TWO functions.
-1. One is send the dictionary to the server side.
-Serialize the dictionary and support user choose 3 types.
-2. One is create a file in local (folder of "Client Side") and 
-send the content of the file to the server side. 
-Support to input the encrypt key to encrypt the file's content and save it on the client side or 
-save it in a non-encrypted txt file.
-"""
+    This is the client-side coding and handles TWO functions.
```

Comment:
Please ensure the development environment is installed pylint to support PEP8. It will hit many different source codes by Git.